client/my-sites/themes/thanks-modal.jsx

```diff
@@ -61,92 +62,59 @@ const ThanksModal = React.createClass( {
 		};
 	},
 
-	renderWpcomInfo() {
-		const features = translate( "Discover this theme's {{a}}awesome features.{{/a}}", {
-			components: {
-				a: <a href={ this.props.detailsUrl }
-					onClick={ this.onLinkClick( 'features' ) } />
-			}
-		} );
-		const customize = translate( '{{a}}Customize{{/a}} this design.', {
-			components: {
-				a: <a href={ this.props.customizeUrl }
-					onClick={ this.onLinkClick( 'customize' ) } />
-			}
-		} );
+	renderBody() {
 		return (
 			<ul>
 				<li>
-					{ this.props.source === 'list' ? features : customize }
+					{ this.props.source === 'list' ? this.renderThemeInfo() : this.renderCustomizeInfo() }
 				</li>
 			<li>
-				{ translate( 'Have questions? Stop by our {{a}}support forums.{{/a}}', {
-					components: {
-						a: <a href={ this.props.forumUrl }
-							onClick={ this.onLinkClick( 'support' ) } />
-					}
-				} ) }
+				{ this.renderSupportInfo() }
 			</li>
 			</ul>
 		);
 	},
 
-	renderWporgThemeInfo( themeUri ) {
-		if ( themeUri ) {
-			return (
-				<li>
-					{ translate( 'Learn more about this {{a}}awesome theme{{/a}}.', {
-						components: {
-							a: <a href={ themeUri }
-								onClick={ this.onLinkClick( 'org theme' ) } />
-						}
-					} ) }
-				</li>
-			);
-		}
+	renderThemeInfo() {
+		return translate( '{{a}}Learn more about{{/a}} this theme.', {
+			components: {
+				a: <a href={ this.props.detailsUrl }
+					onClick={ this.onLinkClick( 'theme info' ) } />
+			}
+		} );
 	},
 
-	renderWporgAuthorInfo( authorUri ) {
-		if ( authorUri ) {
-			return (
-				<li>
-					{ translate( 'Have questions? {{a}}Contact the theme author.{{/a}}', {
-						components: {
-							a: <a href={ authorUri }
-								onClick={ this.onLinkClick( 'org author' ) } />
-						}
-					} ) }
-				</li>
-			);
-		}
+	renderCustomizeInfo() {
+		return translate( '{{a}}Customize{{/a}} this design.', {
+			components: {
+				a: <a href={ this.props.customizeUrl }
+					onClick={ this.onLinkClick( 'customize' ) } />
+			}
+		} );
 	},
 
-	renderWporgForumInfo() {
-		return (
-			<li>
-				{ translate( 'If you need support, visit the WordPress.org {{a}}Themes forum{{/a}}.', {
-					components: {
-						a: <a href="https://wordpress.org/support/forum/themes-and-templates"
-							onClick={ this.onLinkClick( 'org forum' ) } />
-					}
-				} ) }
-			</li>
-		);
-	},
+	renderSupportInfo() {
+		const { author_uri: authorUri } = this.props.currentTheme;
 
-	renderJetpackInfo() {
-		const {
-			theme_uri: themeUri,
-			author_uri: authorUri
-		} = this.props.currentTheme;
+		if ( this.props.forumUrl ) {
+			return translate( 'Have questions? Stop by our {{a}}support forums{{/a}}.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Have questions? Stop by our {{a}}support forums.{{/a}}' )` *ES Score: 16.56*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).